src/etc/rustup.sh

```diff
@@ -564,9 +565,12 @@ install_packages() {
 
     RUST_LOCAL_TARBALL="${RUST_DOWNLOAD_DIR}/${RUST_TARBALL_NAME}"
 
-    download_package \
-        "${RUST_URL}/${RUST_TARBALL_NAME}" \
-        "${RUST_LOCAL_TARBALL}"
+    if [ "${CFG_SKIP_DOWNLOAD}" != 1 ]
+    then
+        download_package \
+            "${RUST_URL}/${RUST_TARBALL_NAME}" \
+            "${RUST_LOCAL_TARBALL}"
+    fi
```

Comment:
could you add an `else` statement and check whether `"${RUST_LOCAL_TARBALL}"` exists and fail otherwise?

This should probably go into `install_package`, though.